code/core/src/csf-tools/ConfigFile.ts

```diff
@@ -736,17 +736,24 @@ export class ConfigFile {
    * @param fromImport - The module to import from
    */
   setRequireImport(importSpecifier: string[] | string, fromImport: string) {
-    const requireDeclaration = this._ast.program.body.find(
-      (node) =>
+    const requireDeclaration = this._ast.program.body.find((node) => {
+      const hasDeclaration =
         t.isVariableDeclaration(node) &&
         node.declarations.length === 1 &&
         t.isVariableDeclarator(node.declarations[0]) &&
         t.isCallExpression(node.declarations[0].init) &&
         t.isIdentifier(node.declarations[0].init.callee) &&
         node.declarations[0].init.callee.name === 'require' &&
         t.isStringLiteral(node.declarations[0].init.arguments[0]) &&
-        node.declarations[0].init.arguments[0].value === fromImport
-    ) as t.VariableDeclaration | undefined;
+        (node.declarations[0].init.arguments[0].value === fromImport ||
+          node.declarations[0].init.arguments[0].value === fromImport.split('node:')[1]);
+      if (hasDeclaration) {
```

Comment:
If you do it like this in the require logic, perhaps do the same in the import logic, then remove lines 868-873 and alter lines 921-923 to be:
```ts
const importDeclaration = this._ast.program.body.find(
  (node) => {
    const hasDeclaration = t.isImportDeclaration(node) && (node.source.value === fromImport || node.source.value === fromImport.split('node:')[1])

    if (hasDeclaration) {
      fromImport = node.source.value;
    }

    return hasDeclaration;
  }
) as t.ImportDeclaration | undefined;
```

In that part, I see that this code is the first one that refers to the `fromImport` variable so it should be safe